src/Model/System/Config/Backend/FetchTagLimit.php

```diff
@@ -0,0 +1,73 @@
+<?php
+declare(strict_types=1);
+
+namespace IntegerNet\AsyncVarnish\Model\System\Config\Backend;
+
+use Magento\Framework\App\Cache\TypeListInterface;
+use Magento\Framework\App\Config\Value;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Framework\Escaper;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Model\Context;
+use Magento\Framework\Model\ResourceModel\AbstractResource;
+use Magento\Framework\Registry;
+
+/**
+ * Backend model for processing Varnish Fetch Tag Limit settings
+ *
+ * Class HeaderLength
+ */
+class FetchTagLimit extends Value
+{
+    /**
+     * @var Escaper
+     */
+    private $escaper;
+
+    /**
+     * Ttl constructor.
+     * @param Context $context
+     * @param Registry $registry
+     * @param ScopeConfigInterface $config
+     * @param TypeListInterface $cacheTypeList
+     * @param AbstractResource|null $resource
+     * @param AbstractDb|null $resourceCollection
+     * @param array $data
+     * @param Escaper|null $escaper
+     */
+    public function __construct(
+        Context $context,
+        Registry $registry,
+        ScopeConfigInterface $config,
+        TypeListInterface $cacheTypeList,
+        ?AbstractResource $resource = null,
+        ?AbstractDb $resourceCollection = null,
+        array $data = [],
+        ?Escaper $escaper = null
+    ) {
+        parent::__construct($context, $registry, $config, $cacheTypeList, $resource, $resourceCollection, $data);
+        $this->escaper = $escaper ?: ObjectManager::getInstance()->create(Escaper::class);
```

Comment:
there is no `escaper` argument? this looks like the pattern of core classes that add new dependencies in a backwards compatible way. but here's a new class, so it can be a regular constructor argument

Reply:
actually, `this->escaper` is never used, so the complete constructor can be deleted

Reply:
It's for backwards compatibility and I don't know in what version the escaper class was introduced. So you want to risk compatibility and just remove it?

Also, it's used. see line 67.

Reply:
It's no longer used in newer Core classes, so I've removed it.